MAP/src/components/StatsBox/StatsBox.js

```diff
@@ -1,19 +1,20 @@
 import React, { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { fetchTotals } from '../../services';
-import './Totals.css';
+import './StatsBox.css';
 import { toBool } from '../../utils/utils';
 
-const separateNumber = (number) => {
-  const nfObject = new Intl.NumberFormat('es-ES');
+const separateNumber = (number, format) => {
+  const nfObject = new Intl.NumberFormat(format);
```

Comment:
I like this change, should definitely keep it. However when I switch languages, the `,` doesn't change to `.` as I would expect